### PR TITLE
Remove antler-proj

### LINKED ISSUE #206: Unlink Antler From Build
## Clean up and remove Antler features

```[tasklist]
### Tasks
- [ ] Antler Proj in https://github.com/AntelopeIO/cdt/pull/114
- [ ] Antler Run in https://github.com/AntelopeIO/cdt/pull/94
- [ ] CDT Macros related to Antler as specified in https://github.com/AntelopeIO/cdt/pull/87
```

### PR DESCRIPTION
**Pull request by dimas1185**
removing antler-proj
resolves #206 

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Review by ericpassmore — APPROVED** [review]
LGTM

File: scripts/generate_tarball.sh [hunk 1]
```diff
@@ -64,8 +64,6 @@ create_symlink eosio-pp eosio-pp
 create_symlink cdt-init cdt-init
 create_symlink eosio-wasm2wast eosio-wasm2wast
 create_symlink eosio-wast2wasm eosio-wast2wasm
-create_symlink eosio-wasm2wast cdt-wasm2wast
-create_symlink eosio-wast2wasm cdt-wast2wasm
```
Comment (arhag) [inline scripts/generate_tarball.sh:68]: Why are these two symlinks removed?

**Review by arhag — CHANGES_REQUESTED** [review]
If this is meant to be in CDT 4.0.1 as the linked issue milestone suggests, then this PR should be closed and a new one targeting `release/4.0` should be opened. Make sure to add the `[4.0]` prefix to the title of the new PR as well.

File: scripts/generate_tarball.sh (hunk 1 above)
Comment (dimas1185) [reply scripts/generate_tarball.sh:68]: yea, should not do this. probably an error in a reverted commit.

**Comment by dimas1185** [comment]
will get to main later after merge from `release/4.0`. corresponding `release/4.0` PR - https://github.com/AntelopeIO/cdt/pull/238

**Pull request closed without merging**